src/99-appendices/05-units.md

```diff
@@ -54,15 +83,15 @@ Following the [International System of Units](https://en.wikipedia.org/wiki/Inte
 
 ### Submultiples
 
-| Prefix name                                 | Prefix symbol | Factor           |
-| ------------------------------------------------------------------ | ---------------------- | ------------------------------ |
-| [deci](https://www.wikiwand.com/en/Deci-)   | d             | 10<sup>-1</sup>  |
-| [centi](https://www.wikiwand.com/en/Centi-) | c             | 10<sup>-2</sup>  |
-| [milli](https://www.wikiwand.com/en/Milli-) | m             | 10<sup>-3</sup>  |
-| [micro](https://www.wikiwand.com/en/Micro-) | µ             | 10<sup>-6</sup>  |
-| [nano](https://www.wikiwand.com/en/Nano-)   | n             | 10<sup>-9</sup>  |
-| [pico](https://www.wikiwand.com/en/Pico-)   | p             | 10<sup>-12</sup> |
-| [femto](https://www.wikiwand.com/en/Femto-) | f             | 10<sup>-15</sup> |
-| [atto](https://www.wikiwand.com/en/Atto-)   | a             | 10<sup>-18</sup> |
-| [zepto](https://www.wikiwand.com/en/Zepto-) | z             | 10<sup>-21</sup> |
-| [yocto](https://www.wikiwand.com/en/Yocto-) | y             | 10<sup>-24</sup> |
+| Prefix name                                 | Prefix symbol   | Factor           |
+| ------------------------------------------- | --------------- | ---------------- |
+| [deci](https://www.wikiwand.com/en/Deci-)   | d               | 10<sup>-1</sup>  |
+| [centi](https://www.wikiwand.com/en/Centi-) | c               | 10<sup>-2</sup>  |
+| [milli](https://www.wikiwand.com/en/Milli-) | m               | 10<sup>-3</sup>  |
+| [micro](https://www.wikiwand.com/en/Micro-) | u or µ (U+00B5) | 10<sup>-6</sup>  |
```

Comment:
why don't we allow both micro and mu symbols? They are indistinguishable to the naked eye and it's mostly [a historical artifact](https://en.wikipedia.org/wiki/Micro-#Symbol_encoding_in_character_sets). I think we should make it easy  for people to adapt BIDS, not make them spend hours understanding the difference between these two symbols.

Reply:
> I think we should make it easy for people to adapt BIDS, not make them spend hours understanding the difference between these two symbols.

I agree. I guess we are balancing between narrowing down possibilities of formatting and being overly pedantic. In this case it's currently perhaps being overly pedantic 🙂 

@effigies brought this up in an online meeting not too long ago ... soo I'll update this soon, thanks @jasmainak

Reply:
The reason that we didn't allow both was that it shifts more of the responsibility onto apps to parse the units. If we can say they're always MIXF, then all you have to know if MIXF. We had some suggestions for micro-, degree and Ohm (IIRC) that made this difficult by backwards compatibility, so we said MIXF + those three code points. We can just as easily say MIXF + mu + micro + degree + Ohm, but there will be a complexity cost to any software that cares about units.

Right now we're seeing a case where the validator requiring micro instead of mu is causing user confusion. We can relieve this momentarily by relaxing the constraint, but will we not simply be distributing the confusion to every tool's bug tracker?

Reply:
humm ... I just went back and checked how we are dealing with this in MNE. We coerce the mu and micro into micro -- so it might be a tool-to-tool issue. But I agree that keeping one would be simpler for tool developers in the long run.

Reply:
I'm with @jasmainak on this one -- the difference between these two characters is imperceptible, so I really don't like pushing this onto users.

I do understand your concern, though, @effigies. If we want to enforce this with the validator, then I think this case of mu vs micro needs a very specific message alerting users to exactly this difference. To my mind, that seems the only reasonable (humane ?) way to handle it.

What I don't know, though, is how easy this would be to accomplish with the current validator structure. WDYT ?

Reply:
I think it should be easily possible in the current validator structure:
- implement a new issue code Z to signal "you are using µ as a character, please be aware that ... do not use U+03BC ..." 

and then for validation:

- when parsing a `unit` column in a TSV file X
- upon encountering unicode `U+03BC` in a line Y of file X
- push an issue to issue list for file X at line Y: issue code Z

what's more of a challenge (and one of the reasons why this PR is stagnating) is to validate MIXF units as a whole :confused:

Reply:
re: validator -> @satra now has implemented a nice Python validator: https://github.com/sensein/cmixf

the JS version of a cmixf validator may follow after merging this PR, but it shouldn't be a hard blocker.

Reply:
we should raise an issue and track these somewhere nonetheless

Reply:
agreed, I am now tracking our discussion here: https://github.com/bids-standard/bids-validator/issues/969

thanks for the many points.

Reply:
Why not rely on Unicode normalization to solve the visually similar character issue? For example, the compatibility decomposition will map U+00B5 to U+03BC. Python and JavaScript both have support for these mappings in the standard libraries. bids-validator could just normalize all Unicode text before validation. The specification could also recommend which normalization is preferred for other software aiming to be compatible.

Reply:
@nellh thanks for your input and also the references (above) -> could you make a concrete proposal what needs to be changeds in this PR (using GitHub's code suggestion feature)?

Reply:
I'd like to point out that according to the unicode standard ([PDF](https://www.unicode.org/reports/tr25/), p.11) the sign `U+03BC` is the preferred symbol. So it'd be surprising to some users to get a warning when using the preferred `U+03BC` instead of `U+00B5` (which according to unicode, should be recognized as well)

Reply:
Thanks @gpiantoni. I take your post as voting for option 3 in my summary (https://github.com/bids-standard/bids-specification/pull/411#issuecomment-633992688), right? ... what are your thoughts about option 1?

Reply:
Thanks for asking my opinion. As end user, I'd prefer option 3. But I understand the complexity of implementing it, so option 1 is probably easier to implement (sorry, I don't know more than this to comment further). Option 2 however would put the BIDS guidelines in disagreement with the unicode standard.